HouseRules_Essentials/Rules/CourageShantyAddsHPRule.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using Boardgame;
+    using Boardgame.BoardEntities;
+    using Boardgame.GameplayEffects;
+    using HarmonyLib;
+    using HouseRules.Types;
+
+    public sealed class CourageShantyAddsHPRule : Rule, IConfigWritable<int>, IPatchable, IMultiplayerSafe
```

Comment:
nit: `CourageShantyAddsHPRule` -> `CourageShantyAddsHpRule` for pascal casing.  As per https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces.

(except we do break this rule already with our `HR` class)